cg/cli/demultiplex/finish.py

```diff
@@ -28,13 +27,18 @@ def post_process_illumina_run(context: CGConfig, demultiplexed_run_dir_name: str
     """Command to post-process an Illumina run after demultiplexing.
 
     demultiplexed-run-dir-name is the full run name, e.g. '230912_A00187_1009_AHK33MDRX3'.
+
     """
-    post_processing_service = IlluminaPostProcessingService(
+
+    factory = PostProcessServiceFactory(
```

Comment:
I think the post-processing should not have a factory, what could have a factory is a time service